auth/client/pom.xml

```diff
@@ -142,11 +142,6 @@
             <version>${version.org.jboss.ws.jbossws-spi}</version>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>org.kohsuke.metainf-services</groupId>
-            <artifactId>metainf-services</artifactId>
-            <scope>provided</scope>
-        </dependency>
```

Comment:
@Skyllarr Just wondering why this is being removed. It looks like it was added for RESTEasy integration:

https://github.com/wildfly-security/wildfly-elytron/pull/1340

Reply:
@fjuma it maybe does not belong to this PR but I noticed it is a double dependency. The same dependency is in this file on line 118.

Reply:
Thanks @Skyllarr, I didn't realize it was a duplicate.